GVFS/GVFS.NativeHooks.Common/common.h

```diff
@@ -1,5 +1,7 @@
 #pragma once
 
+#include <cstring>
+
 #ifdef __APPLE__
```

Comment:
Right; the `#elif` clause for Linux is in another commit in the [`linux-gvfs-build`](https://github.com/Microsoft/VFSForGit/pull/986/commits/de6b32a0270d433b20142d0607aa4a7bb516aebf) branch.

This change to add the `#include <cstring>` here is because there are calls to `strcmp()` and friends in the various hook programs' [`main.cpp`](https://github.com/Microsoft/VFSForGit/blob/master/GVFS/GVFS.PostIndexChangedHook/main.cpp#L18) files, and at least with my GCC compiler these fail unless `<cstring>` is included; it has nothing to do with the various string calls in `common.posix.cpp` elsewhere in this PR.